SU-SWS/decanter-react
# ADAPT-1771: Linting setup.

# READY FOR REVIEW

# Summary
- Sets and fixes linting issues 
- Uses prettier and airbnb standards
- Works in code climate

# Needed By (Date)
- Whenever but not too much longer as these changes will conflict at some point.

# Urgency
- 0

# Steps to Test

1. Review code changes, particularly around the eslint config
2. Review build site to ensure that it still functions and looks as it should

# Affected Projects or Products
- Give, Alumni

# Associated Issues and/or People
- ADAPT-1771

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
